Userland/Libraries/LibCrypto/BigInt/UnsignedBigInteger.h

```diff
@@ -58,7 +59,14 @@ class UnsignedBigInteger {
     [[nodiscard]] String to_base(u16 N) const;
 
     [[nodiscard]] u64 to_u64() const;
-    [[nodiscard]] double to_double() const;
+
+    enum class RoundingMode {
+        IEEERoundAndTiesToEvenMantissa,
+        RoundTowardZero,
+        ECMAScriptNumberValueFor = IEEERoundAndTiesToEvenMantissa,
```

Comment:
My brain might be fried, but `ECMAScriptNumberValueFor` read a bit awkwardly to me ending with "For"? Makes this think "for what?"

Reply:
I thought the same at first and even commented a suggestion to change it, but from a spec perspective it makes sense as that's what they call the operation: https://tc39.es/ecma262/#number-value-for

Happy to see it changed it someone can come up with a better name, but also happy to keep as is.

Reply:
![image](https://user-images.githubusercontent.com/19366641/186920733-8f8758b5-59c2-4331-bf91-eb2bf12154b4.png)

Reply:
It is supposed to be for the value of the big int. Used like
`...to_double(Crypto::UnsignedBigInteger::RoundingMode::ECMAScriptNumberValueFor))` but it is a bit awkward. In the spec they also call it: [Number value for](https://tc39.es/ecma262/#number-value-for)

If you have a nicer suggestion I'm fine with changing it, I'll just let the CI finish first.

Reply:
> Got it, maybe let's drop that spec link against the value name so it's clear?

Yeah that would help, good point. I'll add it